README.md

```diff
@@ -15,6 +24,42 @@ The steps below describe what needs to be done to use Platoon for
 data-parallelism. The LSTM example in the folder 'example' was implemented
 following these steps and should be referred to for guidance.
 
+Timing of the LSTM example on 2 k80
+-----------------------------------
+
+The timing is about efficiency of computation, not efficiency of
+training.  So the parameter alpha is constant. The number of minibatch
+is fixed as the hyper-parameter. The sync is also fixed to be after 10
+mini-batch of computation.
+
+With 1 worker, it won't train well. This isn't recommended. This is
+there just to show the overhead of the EASGD implementation.  Normal
+is without this framework, also there for overhead evaluation.
```

Comment:
Is normal with EASGD or just SGD? If I compare Normal with 1 GPU, is there only the Platoon overhead or also the EASGD overhead?